### PR TITLE
Update links now that PyLXD and LXD have moved to Canonical

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: integration/test_storage.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def test_put(self):
         self.assertEqual(p.description, new_desc)
 
     # can't test this as patch doesn't seem to work for storage pools.
-    # Need to wait until bug: https://github.com/lxc/lxd/issues/4709
+    # Need to wait until bug: https://github.com/canonical/lxd/issues/4709
```
Comment (markylaing) [inline integration/test_storage.py:96]: Looks like this has been fixed? Maybe a follow up PR can unskip this test :smile:

File: pylxd/models/instance.py [hunk 2]
```diff
@@ -853,7 +853,7 @@ def publish(self, public=False, wait=False):
         If wait=True, an Image is returned.
 
         This functionality is currently broken in LXD. Please see
-        https://github.com/lxc/lxd/issues/2201 - The implementation
+        https://github.com/canonical/lxd/issues/2201 - The implementation
```
Comment (markylaing) [inline pylxd/models/instance.py:856]: Looks like this one has also been fixed.

**Review by markylaing — COMMENTED** [review]
Changes look good :+1:

Looks like there are a couple of TODOs that were waiting on LXD issues that can now be addressed.

File: integration/test_storage.py (hunk 1 above)
Comment (simondeziel) [reply integration/test_storage.py:96]: Yep, those were meant to be addressed in a subsequent PR ;)

File: pylxd/models/instance.py (hunk 2 above)
Comment (simondeziel) [reply pylxd/models/instance.py:856]: Same here

File: integration/test_storage.py (hunk 1 above)
Comment (markylaing) [reply integration/test_storage.py:96]: Okey dokey!

**Review by markylaing — APPROVED** [review]
LGTM

**Review by markylaing — APPROVED** [review]

**Pull request merged**